drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -103,6 +103,9 @@ static inline int stm32_clock_control_on(const struct device *dev,
 	case STM32_CLOCK_BUS_AHB2:
 		LL_AHB2_GRP1_EnableClock(pclken->enr);
 		break;
+	case STM32_CLOCK_BUS_AHB3:
+		LL_AHB3_GRP1_EnableClock(pclken->enr);
```

Comment:
Thanks for this addition.
Would you mind adding the equivalent in `stm32_clock_control_off` and `stm32_clock_control_get_subsys_rate`?